### PR TITLE
modules: scrapers: Fix Translator for non-Latin textx

### PR DESCRIPTION
**Pull request by zakaryan2004**
Signed-off-by: Gegham Zakaryan <zakaryan.2004@outlook.com>

### CODE REVIEW

**Comment by raphielscape** [comment]
Test pass?

**Comment by baalajimaestro** [comment]
@raphielscape CI exists for that

File: userbot/modules/scrapers.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 from googleapiclient.errors import HttpError
 from googletrans import LANGUAGES, Translator
 from gtts import gTTS
+import emoji
```
Comment (raphielscape) [inline userbot/modules/scrapers.py]: no full import pls

**Comment by raphielscape** [comment]
> @raphielscape CI exists for that

No I mean the real world test xd

File: userbot/modules/scrapers.py (hunk 1 above)
Comment (zakaryan2004) [reply userbot/modules/scrapers.py]: It's a very small library, but will try to make the imports more narrowed.

**Comment by zakaryan2004** [comment]
> Test pass?

Yeah, I have tested.

File: userbot/modules/scrapers.py (hunk 1 above)
Comment (raphielscape) [reply userbot/modules/scrapers.py]: Ok

**Comment by zakaryan2004** [comment]
@raphielscape narrowed the import. Now it can be merged.

**Comment by baalajimaestro** [comment]
@zakaryan2004 Kek okay.

**Pull request merged**